pymatgen/vis/structure_vtk.py

```diff
@@ -629,11 +629,11 @@ def add_faces(self, faces, color, opacity=0.35):
             else:
                 raise ValueError("Number of points for a face should be >= 3")
 
-    def add_edges(self, edges, type="line", linewidth=2, color=(0.0, 0.0, 0.0)):
+    def add_edges(self, edges, type="line", linewidth=2, color=(0.0, 0.0, 0.0)):  # noqa: A002
```

Comment:
Does this unused `type` arg exist for a reason? If not, removing it would get rid of this `A002` error

Reply:
i'd guess it's there to conform with the signature of the parent class?

forgot to mention earlier, `ruff` [can auto-insert](https://docs.astral.sh/ruff/tutorial/#adding-rules) `noqa`s with the ``--add-noqa` flag

Reply:
> i'd guess it's there to conform with the signature of the parent class?

Not sure, this looks like the base class: https://github.com/materialsproject/pymatgen/blob/624e6eb5cfefbf5736fe31b8910dd19c1042e94e/pymatgen/vis/structure_vtk.py#L35

> forgot to mention earlier, `ruff` [can auto-insert](https://docs.astral.sh/ruff/tutorial/#adding-rules) `noqa`s with the `--add-noqa` flag

Thanks for the tip!